timeseries/src/autogluon/timeseries/models/abstract/abstract_timeseries_model.py

```diff
@@ -232,6 +381,18 @@ def get_params(self) -> dict:
             target=self.target,
         )
 
+    @classmethod
+    def load_info(cls, path: str, load_model_if_required: bool = True) -> dict:
```

Comment:
Do we actually save or load info in TS models? Looking at existing trained predictors, we never save `info.pkl`.

Reply:
will refactor.